### PR TITLE
rename MsgIntersectUnchanged to MsgIntersectNotFound

### PR DESCRIPTION
**Pull request by MarcFontaine**
Rename `MsgIntersectUnchanged` to `MsgIntersectNotFound`
including the identifiers that are derived from the message name,
documentation and wireshark-plugin.

https://github.com/input-output-hk/ouroboros-network/issues/849

This requires updates to cardano-node but only two lines

### CODE REVIEW

File: ouroboros-network/src/Ouroboros/Network/Protocol/ChainSync/Test.hs [hunk 1]
```diff
@@ -184,8 +184,8 @@ instance (Eq header, Eq point) => Eq (AnyMessage (ChainSync header point)) where
   AnyMessage (MsgRollForward h1 p1)       == AnyMessage (MsgRollForward h2 p2)       = h1 == h2 && p1 == p2
   AnyMessage (MsgRollBackward p1 h1)      == AnyMessage (MsgRollBackward p2 h2)      = p1 == p2 && h1 == h2
   AnyMessage (MsgFindIntersect ps1)       == AnyMessage (MsgFindIntersect ps2)       = ps1 == ps2
-  AnyMessage (MsgIntersectFound p1 h1)    == AnyMessage (MsgIntersectFound p2 h2) = p1 == p2 && h1 == h2
-  AnyMessage (MsgIntersectUnchanged h1)   == AnyMessage (MsgIntersectUnchanged h2)   = h1 == h2
+  AnyMessage (MsgIntersectFound p1 h1)    == AnyMessage (MsgIntersectFound p2 h2)    = p1 == p2 && h1 == h2
+  AnyMessage (MsgIntersectNotFound h1)    == AnyMessage (MsgIntersectNotFound h2)    = h1 == h2
```
Comment (coot) [inline ouroboros-network/src/Ouroboros/Network/Protocol/ChainSync/Test.hs]: Could you remove a few spaces of indentation before the `=` sign.

File: ouroboros-network/src/Ouroboros/Network/Protocol/ChainSync/Direct.hs [hunk 2]
```diff
@@ -42,14 +42,14 @@ direct_  ServerStIdle{recvMsgRequestNext}
...
-                            recvMsgIntersectUnchanged}) = do
+                            recvMsgIntersectNotFound}) = do
     sIntersect <- recvMsgFindIntersect points
     case sIntersect of
       SendMsgIntersectFound  pIntersect pHead server' ->
         direct server' (recvMsgIntersectFound pIntersect pHead)
 
-      SendMsgIntersectUnchanged            pHead server' ->
-        direct server' (recvMsgIntersectUnchanged pHead)
+      SendMsgIntersectNotFound            pHead server' ->
```
Comment (coot) [inline ouroboros-network/src/Ouroboros/Network/Protocol/ChainSync/Direct.hs]: The indentation got off at some point here, could you align the `pHead` to the `pHead` just above.

File: ouroboros-network/src/Ouroboros/Network/Protocol/ChainSync/Test.hs (hunk 1 above)
Comment (MarcFontaine) [reply ouroboros-network/src/Ouroboros/Network/Protocol/ChainSync/Test.hs]: OK

File: ouroboros-network/src/Ouroboros/Network/Protocol/ChainSync/Direct.hs (hunk 2 above)
Comment (MarcFontaine) [reply ouroboros-network/src/Ouroboros/Network/Protocol/ChainSync/Direct.hs]: OK

**Pull request merged**